R/infrastructure.R

```diff
@@ -33,6 +39,12 @@ use_testthat <- function(pkg = ".") {
     pkg = pkg
   )
 
+  if (use_catch) {
```

Comment:
I think that makes sense -- export `use_catch()` on the `devtools` side, use that function within the `use_testthat()` call when `catch = TRUE`, and (after this is merged) we could deprecate `use_catch()` within `testthat`. Does that make sense, @hadley?